### DISCUSSION TITLE
Choppy playback of merged hevc/x265 video files

### DISCUSSION THREAD

**Discussion by kylealex9** [mifi/lossless-cut#1525] [General]
Hi Mifi,
I haved used Lossless Cut for quite some time now and really enjoyed it; thanks so much! One problem I'm trying to figure out, though, is the choppy playback of merged hevc/x265 video files. The points where those videos are merged do not run smoothly and in some cases the visuals are distorted. I have not experienced this issue with merged videos in other formats (avi, x264). Is there anything I could do to fix this? Thanks!

**Comment by mifi** [comment]
Hi. You can check troubleshooting, and try different options (and advanced options). If none of it helps, then i dont know what helps. You could also try to cut without audio and see if that helps
